core/src/test/scala/integration/kafka/api/SaslSslAdminIntegrationTest.scala

```diff
@@ -360,6 +360,8 @@ class SaslSslAdminIntegrationTest extends BaseAdminIntegrationTest with SaslSetu
 
     // Test that we cannot create or delete ACLs when ALTER is denied.
     authorizationAdmin.addClusterAcl(DENY, ALTER)
+    // CLUSTER_ACTION shall also be forbid since the request goes to privilege listener
+    authorizationAdmin.addClusterAcl(DENY, CLUSTER_ACTION)
```

Comment:
Is this change needed? I am not sure I follow the comment about the privileged listener. That shouldn't affect ACLs I think.